src/components/metadata/styles/metadata.scss

```diff
@@ -262,8 +262,8 @@ $list-inline-spacing: 0.2em;
 
 .pipeline-metadata__plot {
   display: flex;
-  cursor: pointer;
   height: 300px;
+  cursor: pointer;
```

Comment:
I wonder what is causing all these changes in the scss files?

Reply:
I think it's prettier ran through husky.